src/seedu/addressbook/data/person/Block.java

```diff
@@ -0,0 +1,33 @@
+package seedu.addressbook.data.person;
+
+
+public class Block {
+
+    public static final String BLOCK_VALIDATION_REGEX = "\\d+";
+    
+    public final String value;
+
+    
+    public Block(String block) {
+        this.value = block;
+        
+    }
+
+
+    /**
+     * Returns true if a given string is a valid person block.
+     */
+    public static boolean isValidBlock(String block) {
```

Comment:
Don't see the need for this, as the method is never called. The address is already validated using the `isValidAddress` function